### DISCUSSION TITLE
Games vs. Sandboxie Plus

### DISCUSSION THREAD

**Discussion by Mitch037** [sandboxie-plus/Sandboxie#4172] [Documents and notes]
# Games :vs: Sandboxie Plus
### Compatibility with games and applications
The list features games and applications which have been tested with Sandboxie Plus. The table provides detailed info about each application, environment it was tested in and reported issues. In order to find out any discrepancies, listed applications have been tested in [Standard (![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b)yellow) box](https://sandboxie-plus.com/box-preset-comparison/) and in [Application Compartment (![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68)green) box](https://sandboxie-plus.com/compartment-mode/). Lastly, some applications have been tested in order to find out performance difference between the aforementioned box types.
<hr>

Table legend:
:white_check_mark: – works properly
:x: – doesn't work
:grey_question: – not tested
:repeat: – repetition, the same issue as in the previous box
<hr>:arrow_double_down:APPS

| Title | ![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b) Standard | ![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68) Application Compartment | ![SandMan_IDI_ICON1](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/081e9dec-9e7f-4fdc-a1cc-cf6c2511adcd) Version | :computer: OS
| :---: | :---: | :---: | :---: | :---: |
| [EA App](https://www.ea.com/ea-app/) | :x:<br>#3260 | :x:<br>:repeat: | 1.11.0 | Win7x64
| [Epic Games Launcher](https://store.epicgames.com/download/) | :white_check_mark:<br>#3261 | :white_check_mark:<br>:repeat: | 1.15.4 | Win7x64
| [GeForce NOW](https://www.nvidia.com/en-us/geforce-now/download/) | :white_check_mark: | :white_check_mark: | 1.16.3 | Win7x64
| [GOG Galaxy](https://www.gog.com/galaxy) | :x:<br>#3297<br>#1246 | :white_check_mark: | 1.15.4 | Win7x64
| [Origin](https://en.wikipedia.org/wiki/EA_(service)#Origin_client) | :white_check_mark: | :white_check_mark: | 1.15.4 | Win7x64
| [Steam](https://store.steampowered.com/about/) | :white_check_mark:<br>#3484 | :white_check_mark: | 1.15.4 | Win7x64
| [Ubisoft Connect](https://ubisoftconnect.com/) | :white_check_mark:<br>#4490 | :white_check_mark:<br>:repeat: | 1.15.6 | Win7x64
<br>
:arrow_double_down:GAMES

| Title | ![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b) Standard | ![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68) Application Compartment | ![SandMan_IDI_ICON1](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/081e9dec-9e7f-4fdc-a1cc-cf6c2511adcd) Version | :computer: OS
| :---: | :---: | :---: | :---: | :---: |
| [Alice: Madness Returns](https://www.mobygames.com/game/51996/alice-madness-returns/)<br>• Steam | :grey_question: | :white_check_mark: | 1.15.4 | Win7x64
| [America's Army: Operations](https://www.mobygames.com/game/7640/americas-army-operations/)<br>• AAO25 | :white_check_mark: | :white_check_mark: | 1.15.6 | Win7x64
| [American McGee's Alice](https://www.mobygames.com/game/2703/american-mcgees-alice/)<br>• Remastered | :white_check_mark: | :white_check_mark: | 1.15.4 | Win7x64
| [Amerzone: The Explorer's Legacy](https://www.mobygames.com/game/3973/amerzone-the-explorers-legacy/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Another War](https://www.mobygames.com/game/11667/another-war/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.0 | Win7x64
| [Arma: Cold War Assault](https://www.mobygames.com/game/53252/arma-cold-war-assault/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.2 | Win7x64
| [Battlefield 2](https://www.mobygames.com/game/18194/battlefield-2/)<br>• Retail<br>• BF2Hub | :white_check_mark:<br>#3136<br>:warning: _Block Internet Access_ preset causes the game to crash | :white_check_mark:<br>:repeat:<br>:repeat: | 1.15.8 | Win7x64
| [Battlefield 3](https://www.mobygames.com/game/53433/battlefield-3/)<br>• Origin | :white_check_mark:<br>#3136 | :white_check_mark:<br>:repeat: | 1.14.9 | Win7x64
| [Catan: Die Erste Insel](https://www.mobygames.com/game/16206/catan-die-erste-insel/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Counter-Strike](https://www.mobygames.com/game/165613/counter-strike/)<br>• Steam | :grey_question: | :white_check_mark: | 1.16.0 | Win7x64
| [Cue Club 2](https://www.mobygames.com/game/102939/cue-club-2/)<br>• Steam | :grey_question: | :white_check_mark: | 1.15.8 | Win7x64
| [Day of Defeat](https://www.mobygames.com/game/9240/day-of-defeat/)<br>• Steam | :grey_question: | :white_check_mark: | 1.16.0 | Win7x64
| [Devastation](https://www.mobygames.com/game/8823/devastation/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.9.7 | Win7x64
| [Driver Syndicate](https://soapyman.itch.io/driversyndicate) | :white_check_mark:<br>#4921 | :white_check_mark: | 1.16.0 | Win7x64
| [Driver: San Francisco](https://www.mobygames.com/game/53655/driver-san-francisco/)<br>• Ubisoft Connect | :white_check_mark: | :white_check_mark: | 1.16.0 | Win7x64
| [Fuel](https://www.mobygames.com/game/41124/fuel/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.8 | Win7x64
| [Gorky 02: Aurora Watching](https://www.mobygames.com/game/22093/soldier-elite-zero-hour/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.0 | Win7x64
| [Gorky 17](https://www.mobygames.com/game/2185/odium/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.16.0 | Win7x64
| [Gorky Zero: Beyond Honor](https://www.mobygames.com/game/21105/gorky-zero-beyond-honor/)<br>• Retail | :white_check_mark:<br>:warning: The game EXE file may require setting Compatibility Mode to avoid [infinite loading bug](https://www.pcgamingwiki.com/wiki/Gorky_Zero:_Beyond_Honor#Freeze.2FInfinite_Loading). In such case it should be done via unsandboxed Windows Explorer. | :white_check_mark:<br>:repeat: | 1.16.0 | Win7x64
| [GRID 2](https://www.mobygames.com/game/60730/grid-2/)<br>• Steam | :x:<br>#3218 | :x:<br>:repeat:  | 1.11.0 | Win7x64
| [GRID: Autosport](https://www.mobygames.com/game/72953/grid-autosport/)<br>• Steam | :grey_question: | :white_check_mark: | 1.15.8 | Win7x64
| [Half-Life](https://www.mobygames.com/game/155/half-life/)<br>• Steam | :grey_question: | :white_check_mark: | 1.15.6 | Win7x64
| [Ignite](https://www.mobygames.com/game/76838/ignite/)<br>• Steam | :white_check_mark: | :white_check_mark: | 1.18.1 | Win7x64
| [Infernal](https://www.mobygames.com/game/26739/infernal/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.3 | Win7x64
| [Jazz Jackrabbit 2 Collection](https://www.mobygames.com/game/98842/jazz-jackrabbit-2-collection/)<br>• GOG<br>• JJ2+6.6g | :white_check_mark: | :white_check_mark: | 1.18.1 | Win7x64
| [Kingpin: Life of Crime](https://www.mobygames.com/game/1345/kingpin-life-of-crime/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.15.6 | Win7x64
| [Nosferatu: The Wrath of Malachi](https://www.mobygames.com/game/12130/nosferatu-the-wrath-of-malachi/)<br>• Retail<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Oni](https://www.mobygames.com/game/3242/oni/)<br>• Anniversary Edition | :white_check_mark: | :white_check_mark: | 1.14.6 | Win7x64
| [Operation Flashpoint GOTY](https://www.mobygames.com/game/12132/operation-flashpoint-game-of-the-year-edition/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Pizza Syndicate](https://www.mobygames.com/game/2864/fast-food-tycoon/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Quake III: Gold](https://www.mobygames.com/game/5175/quake-iii-gold/)<br>• GOG<br>• ioquake3<br>• True Combat | :white_check_mark: | :white_check_mark: | 1.15.8 | Win7x64
| [Rainbow Six: Rogue Spear](https://www.mobygames.com/game/964/tom-clancys-rainbow-six-rogue-spear/)<br>• Black Ops 2.0 | :white_check_mark: | :white_check_mark: | 1.15.8 | Win7x64
| [Rainbow Six: Rogue Spear - Black Thorn](https://www.mobygames.com/game/5392/tom-clancys-rainbow-six-rogue-spear-black-thorn/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.15.8 | Win7x64
| [RC Cars](https://www.mobygames.com/game/12554/rc-cars/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Restricted Area](https://www.mobygames.com/game/15479/restricted-area/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.14.9 | Win7x64
| [Starsky & Hutch](https://www.mobygames.com/game/9615/starsky-hutch/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.0 | Win7x64
| [Stunt GP](https://www.mobygames.com/game/5411/stunt-gp/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.8 | Win7x64
| [Söldner: Secret Wars](https://www.mobygames.com/game/15230/soldner-secret-wars/)<br>• Community Edition | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Teenagent](https://www.mobygames.com/game/6423/teen-agent/)<br>• GOG | :white_check_mark: | :white_check_mark: | 1.14.10 | Win7x64
| [Test Drive Unlimited](https://www.mobygames.com/game/26898/test-drive-unlimited/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.15.11 | Win7x64
| [Test Drive Unlimited 2](https://www.mobygames.com/game/51167/test-drive-unlimited-2/)<br>• Retail<br>• Project Paradise 2 | :white_check_mark:<br>:warning: Project Paradise 2 requires HOSTS file redirection in order to work. Set <i>DNS Filter</i> as in the picture available at #5089. | :white_check_mark:<br>:repeat: | 1.15.12 | Win7x64
| [The Sacred Amulet](https://www.mobygames.com/game/3770/the-sacred-amulet/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.11.2 | Win7x64
| [The Simpsons: Hit & Run](https://www.mobygames.com/game/10421/the-simpsons-hit-run/)<br>• Lucas' Simpsons Hit & Run Mod Launcher | :white_check_mark: | :white_check_mark: | 1.16.0 | Win7x64
| [The Sting!](https://www.mobygames.com/game/4295/the-sting/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [TimeScape: Journey to Pompeii](https://www.mobygames.com/game/2482/timescape-journey-to-pompeii/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.1 | Win7x64
| [Traitors Gate](https://www.mobygames.com/game/1863/traitors-gate/)<br>• Retail | :x:<br>:warning: QuickTime (version 2, 3 and 4) doesn't work. It is required to install QuickTime 7.7.9 or QuickTime Alternative 3.2.2<br>:warning: QuickTime videos are laggy<br>• Launches instantly | :white_check_mark:<br>• QuickTime (version 2, 3 and 4) works fine but it is recommended to install QuickTime 7.7.9 or QuickTime Alternative 3.2.2<br>• QuickTime videos work fine<br>:warning: Takes 60s to launch | 1.12.7 | Win7x64
| [Traitors Gate 2](https://www.mobygames.com/game/10965/traitors-gate-2-cypher/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.14.6 | Win7x64
| [Unreal Gold](https://www.mobygames.com/game/4892/unreal-gold/)<br>• Retail<br>• GOG<br>• OldUnreal227j | :white_check_mark: | :white_check_mark: | 1.9.3 | Win7x64
| [Unreal Tournament GOTY](https://www.mobygames.com/game/7158/unreal-tournament-game-of-the-year-edition/)<br>• Retail<br>• OldUnreal-UTPatch469e | :white_check_mark: | :white_check_mark: | 1.15.8 | Win7x64
| [Weird Wars: The Unknown Episode of World War II](https://www.mobygames.com/game/44577/jak-rozpeta%C5%82em-ii-wojne-swiatowa-nieznane-przygody-franka-dolasa/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.4 | Win7x64
| [Wolfenstein: Enemy Territory](https://www.mobygames.com/game/9354/wolfenstein-enemy-territory/)<br>• ET: Legacy | :white_check_mark: | :white_check_mark: | 1.15.8 | Win7x64
| [World in Conflict: Complete Edition](https://www.mobygames.com/game/30129/world-in-conflict/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.16.0 | Win7x64
| [World Racing 2](https://www.mobygames.com/game/22600/world-racing-2/)<br>• Steam | :grey_question: | :white_check_mark: | 1.10.2 | Win7x64
| [WWII Tank Commander](https://www.mobygames.com/game/18686/wwii-tank-commander/)<br>• Retail | :white_check_mark: | :white_check_mark: | 1.10.3 | Win7x64
<br>
:arrow_double_down:BENCHMARK

| Title | ![SbieCtrl_TRAYICON_EMPTY](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/d08ad3c1-c4e3-4a91-a373-475bb0d6726b) Standard | ![sandbox-g-empty](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/07c0361c-ed29-4090-8bcd-5d4d877f1e68) Application Compartment | :open_file_folder: Unsandboxed | ![SandMan_IDI_ICON1](https://github.com/sandboxie-plus/Sandboxie/assets/93145383/081e9dec-9e7f-4fdc-a1cc-cf6c2511adcd) Version | :computer: OS
| :---: | :---: | :---: | :---: | :---: | :---: |
| [GRID 2](https://www.mobygames.com/game/60730/grid-2/)<br>• Steam | min_fps="105.671944"<br>max_fps="159.882614"<br>av_fps="129.760391" | min_fps="137.505951"<br>max_fps="237.419540"<br>av_fps="173.295853  | :grey_question: | 1.12.6 | Win7x64
| [GRID: Autosport](https://www.mobygames.com/game/72953/grid-autosport/)<br>• Steam | min_fps="83.408981"<br>max_fps="113.926949"<br>av_fps="101.507591" | min_fps="115.546089"<br>max_fps="203.296005"<br>av_fps="150.188889"  | :grey_question: | 1.12.6 | Win7x64
| [Operation Flashpoint GOTY](https://www.mobygames.com/game/12132/operation-flashpoint-game-of-the-year-edition/)<br>• GOG | 2449 points | 2548 points | 5034 points | 1.12.7 | Win7x64
| [World in Conflict: Complete Edition](https://www.mobygames.com/game/30129/world-in-conflict/)<br>• Retail | average: 78<br>minimum: 17<br>maximum: 200 | average: 82<br>minimum: 18<br>maximum: 214 | :grey_question: | 1.16.0 | Win7x64

**Comment by minhbac260** [comment]
can you try some Riot game ?

**Comment by e-t-l** [comment]
Has anyone else had issues running games in Steam? I can make Steam itself run fine in a Standard yellow sandbox on Windows 11, but when I try to actually launch a game, SBIE throws a bunch of errors:
```
DXSETUP.exe: SBIE2303 Could not hook CoCreateInstance (4)
DXSETUP.exe: SBIE2318 DLL initialization failed for 'combase.dll'
DXSETUP.exe: SBIE2224 Sandboxed program has crashed: DXSETUP.exe
bms.exe: SBIE2224 Sandboxed program has crashed: bms.exe
```
If this is abnormal I can create a new issue for it, but I was under the belief that Steam should work relatively well with Sandboxie defaults...
(the game, if you want to know, is Black Mesa)

**Reply by e-t-l** [reply]
Ah nevermind, I discovered "bms.exe" was the Black Mesa game executable itself, and "DXSETUP.exe" was a DirectX component installer. I ran it separately and it installed fine, then afterwards I was able to run bms.exe directly and launch the game successfully. No idea what "CoCreateInstance" or "combase.dll" are, but my immediate problem is fixed at least

**Comment by shearercole** [comment]
Latest Steam and Ubisoft Connect client works on standard sandbox - SBIE Plus v1.15.5 - Win 10 Pro 22H2 64-bit

But if Ubisoft Connect ever tries to update itself, it will fail and mess up your install [https://github.com/sandboxie-plus/Sandboxie/issues/4490](url) . Workaround available in issue's comments.

**Comment by shearercole** [comment]
SBIE Plus v1.15.12 - Win 10 Pro 22H2 64-bit 10.0.19045.5737

_**Like A Dragon: Pirate Yakuza in Hawaii (Steam)**_ , It is the latest Yakuza game with Denuvo DRM. Failed to launch in sandbox, black screen at startup for a few secs, then CTD.  No luck with Standard Sandbox & Application Compartment. Unsandboxed works of course.

..... while 2024(and older) Denuvo games from the Yakuza series like _**Like A Dragon: Infinite Wealth(2024)**_ works fine.

**Comment by shearercole** [comment]
SBIE Plus v1.15.12 - Win 10 Pro 22H2 64-bit 10.0.19045.6466

**Metal Gear Solid Delta: Snake Eater 2025 (Steam)** , not working with standard sandbox. During launch, right after shader compilation setup, game's window disappears but process stay idle in Task Mgr.  `OpenWinClass=*` does not help.

These seem to work:
```
NoSysCallHooks=y
NoSandboxieDesktop=y
UnrestrictedToken=y
```

Or Application Compartment sandbox also works , but couldn't test longer than 5 mins cause no supporter cert.

**Comment by love-code-yeyixiao** [comment]
You can try applying for a trial certificate.












YeYixiao
***@***.***






---Original---
From: ***@***.***&gt;
Date: Sun, Dec 7, 2025 19:16 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: Re: [sandboxie-plus/Sandboxie] Games vs. Sandboxie Plus (Discussion#4172)




 
SBIE Plus v1.15.12 - Win 10 Pro 22H2 64-bit 10.0.19045.6466
 
Metal Gear Solid Delta: Snake Eater 2025 (Steam) , not working with standard sandbox. During launch, right after shader compilation setup, game's window disappears but process stay idle in Task Mgr. Works in Application Compartment sandbox but couldn't test longer than 5 mins cause no supporter cert.

**Comment by shearercole** [comment]
SBIE Plus v1.16.9 - Win 10 Pro 22H2 64-bit 10.0.19045.6466

**Monster Hunter Wilds (Steam)** , not working with standard sandbox. Crashes few secs after launching.

Works in Application Compartment sandbox, or by using either of following: `UnrestrictedToken=y` or `UnstrippedToken=y`

**Comment by shearercole** [comment]
SBIE Plus v1.17.2 - Win 10 Pro 22H2 64-bit 10.0.19045.6466

**Stellar Blade v1.4.1 (Steam)** , not working with standard sandbox. Crashes few secs after launching.

Works in Application Compartment sandbox, or by using either of following: `UnrestrictedToken=y` or `UnstrippedToken=y`

**Comment by shearercole** [comment]
SBIE Plus v1.17.9 - Win 10 Pro 22H2 64-bit 10.0.19045.6466

**Yakuza Kiwami 3 & Dark Ties (Steam)**, not working with standard sandbox. Crashes few secs after launching.
**Life is Strange: Reunion (Steam)**, not working with standard sandbox. Crashes few secs after launching.

Both work in Application Compartment sandbox, or by using either of following: `UnrestrictedToken=y` or `UnstrippedToken=y`

**Reply by Mitch037** [reply]
> SBIE Plus v1.17.9 - Win 10 Pro 22H2 64-bit 10.0.19045.6466
> 
> **Yakuza Kiwami 3 & Dark Ties (Steam)**, not working with standard sandbox. Crashes few secs after launching. **Life is Strange: Reunion (Steam)**, not working with standard sandbox. Crashes few secs after launching.
> 
> Both work in Application Compartment sandbox, or by using either of following: `UnrestrictedToken=y` or `UnstrippedToken=y`

It is worth mentioning that both games share [Denuvo protection](https://en.wikipedia.org/wiki/Denuvo).
